### DISCUSSION TITLE
Can i use this firmware for my MKS TFT35 V3.0 ?

### DISCUSSION THREAD

**Discussion by edoluck90** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1422] [Q&A]
hello ,i have a flsun cube with MKS TFT35 V3.0 lcd screen and i  see the customization made possible with this firmware.
So i can use this to update my lcd ?

**Comment by kisslorand** [comment]
No. There's no HAL (hardware abstraction layer) defined for this TFT.
